Drv/LinuxI2CDriver/Events.xml

```diff
@@ -0,0 +1,89 @@
+<events>
+    <!-- Open error -->
+    <event id="0" name="GP_OpenError" severity="WARNING_HI" format_string = "Error opening GPIO device %d: %d (%s)" >
+        <comment>
+        Open error
+        </comment>
+        <args>
+            <arg name="gpio" type="I32">
```

Comment:
You'll need to go through this whole file, and ensure that it is for I2C not GPIO.

Reply:
Ok will do

Reply:
Where can I find documentation for creating an Events.xml file?

Reply:
You are not emitting events, it is best to remove the file.

Reply:
Just delete this file, it is not used.  Nor is it included in your ComponentAi file.